source/manual/govuk-notify.html.md

```diff
@@ -0,0 +1,73 @@
+---
+owner_slack: "#govuk-developers"
+title: How we use GOV.UK Notify
+section: Emails
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-04-29
+review_in: 6 months
+---
+
+[GOV.UK Notify][notify] is as Government-as-a-Platform service that allows
+clients of their API to send emails, text messages and letters. We use GOV.UK
+Notify to send emails to users, both citizens and publishers. Historically
+we've also used AWS SES to send emails, but that's being phased out in favour
+of GOV.UK Notify.
+
+[notify]: https://www.notifications.service.gov.uk/
+
+We have two main services on GOV.UK Notify (six in total as we have a service
+for each environment):
+
+- **GOV.UK Emails**
+
+  This service is used by Email Alert API only. It's used to send citizen-facing
+  email updates about pieces of content on GOV.UK. It's our biggest sender of
+  emails and regularly exceeds one million emails per day.
+
+- **GOV.UK Publishing**
+
+  This service is used by our publishing applications. It's used to send
+  publisher-facing emails. The type of email can vary a lot, but a typical
+  example might be to receive an email when a scheduled document on GOV.UK has
+  been automatically published.
+
+_In the future we may set up a new 'GOV.UK Public' service which is used to
+send citizen-facing emails which haven't gone through Email Alert API._
+
+## Accessing the dashboard
+
+**[👉 Sign in to the GOV.UK Notify dashboard](https://www.notifications.service.gov.uk/sign-in)**
+
+You can either use your own credentials (if you have them) or you can use the
+credentials in [govuk-secrets].
+
+[govuk-secrets]: https://github.com/alphagov/govuk-secrets
```

Comment:
Minor: Alias links are usually grouped together 'reference style', at the end of the page or section. I don't see much of an advantage to making them aliased if they're just dropped in below the sentence they're used.

Reply:
Kind of similar to above, the advantage I see of the aliased links is mostly to improve the readability of the paragraph while reading it as Markdown, and I prefer to keep the aliased links relatively close to where they are used so you don't need to scroll too much to find the links.

Reply:
I guess we don't have a markdown style guide, so we're all going to develop our own preferences 😅